src/components/Booklist.js

```diff
@@ -0,0 +1,11 @@
+import Book from './Book';
+import AddForm from './AddForm';
+
+const Booklist = () => (
```

Comment:
- [ ] The books should be rendered by the "BookLIst" component, using the new "Book" component from the previous step for the book data (props).

## Explanation and hints:

### On your BooksList main component, just use `map` to pass all the props to every `Book` and render them like this example:

```JAVASCRIPT
import Book from '../Book.js';

const BookList = () => {

const booksData = [ // added books here just to ilustrate
    { id: 1, title: 'The Hobbit', author: 'J.R.R. Tolkien' },
    { id: 2, title: 'The Da Vinci Code', author: 'Dan Brown' },
    { id: 3, title: 'Pinocchio', author: 'Carlo Collodi' },
  ];

const bookItems = booksData.map((element) => (
    <Book
      key={element.id}
      id={element.id}
      genre={element.genre}
      title={element.title}
      author={element.author}
      percentageCompleted={element.percentageCompleted}
      currentChapter={element.currentChapter}
    />
  ));

  return (
    <>
      {bookItems}
      <Form />
    </>
  );
}

export default BookList;
```

I hope this is helpful.
That way, your project will be ready for the data and for the next step!! 💪 

_(These are just examples of one way to do it based on my own experience and effort. Included them here just to help and give you an idea, and should not be taken as an "official" way to do it. For this milestone, you only need the title and the author._

Reply:
This was very helpful, thank you so much